addons/mail/static/src/core/common/composer.js

```diff
@@ -713,7 +705,9 @@ export class Composer extends Component {
         const secondPart = text.slice(composer.selection.end, text.length);
         composer.text = firstPart + str + secondPart;
         this.selection.moveCursor((firstPart + str).length);
-        if (!this.ui.isSmall) {
+        if (this.ui.isSmall) {
+            return false;
+        } else {
```

Comment:
I don't get this change, what's the point of this `return false`?

Reply:
It's to not close the picker in mobile. This makes it more natural to use picker in mobile and matches the behaviour of our competitors.

Reply:
It was already the case before no? This `false` doesn't look useful when I read the code.